doc/man7/Ed25519.pod

```diff
@@ -0,0 +1,67 @@
+=pod
+
+=head1 NAME
+
+Ed25519 - EVP_PKEY Ed25519 support
+
+=head1 DESCRIPTION
+
+The B<Ed25519> EVP_PKEY implementation supports key generation, one shot
+digest sign and digest verify using PureEdDSA and B<Ed25519>. It has
```

Comment:
It's probably worth mentioning RFC 8032 here, to give the reader a hint as to what "Pure"EdDSA is and how it relates to the "EdDSA" that they are more likely (?) to have heard of.